sdks/python/apache_beam/options/pipeline_options.py

```diff
@@ -374,6 +374,13 @@ def _add_argparse_args(cls, parser):
     parser.add_argument('--template_location',
                         default=None,
                         help='Save job to specified local or GCS location.')
+    parser.add_argument(
+        '--label', '--labels',
+        dest='labels',
+        action='append',
+        default=None,
+        help='Labels that will be applied to the billing records for this job. '
```

Comment:
<!--new_thread; commit:a4e36ef94c4f7a26c96e424c18408b5ae42ca16d; resolved:0-->
Is "labels" a general purpose Dataflow concept, or just for billing?